data/darktableconfig.xml.in

```diff
@@ -923,11 +923,11 @@
     <longdescription>only select images that have not already been imported</longdescription>
   </dtconfig>
   <dtconfig ui="yes">
-    <name>ui_last/import_ignore_jpegs</name>
+    <name>ui_last/import_ignore_nonraws</name>
     <type>bool</type>
     <default>false</default>
-    <shortdescription>ignore JPEG images</shortdescription>
-    <longdescription>when having raw+JPEG images together in one directory it makes no sense to import both. with this flag one can ignore all JPEGs found.</longdescription>
+    <shortdescription>ignore non-raw images</shortdescription>
+    <longdescription>you may want to import only raws when paired raw and non-raw images are placed together in the same directory. all non-raws found can be ignored with this flag.</longdescription>
```

Comment:
This somewhat suggests that _only_ jpegs that are _paired_ with raws will be ignored. So that if (accidentally) some shots are jpeg only, they would _not_ be weeded out and lost completely (this might be nice, or even expected, functionality). I don't believe this is what is intended or provided (I had to double check!), so this might be more carefully worded.

Reply:
The new text is actually a rewording of the old one, replacing the reference to a specific JPEG format with "non-raw"

Therefore, if the new tooltip contains, in your opinion, a certain ambiguity, the same was true in the previous text.

> so this might be more carefully worded.

Yes, it might be. Please suggest your version of the text.

Reply:
> > replacing the reference to a specific JPEG format with "non-raw"
> 
> And adding the word "paired", which in my mind creates the possible confusion.

And before it was "raw+JPEG", where plus had the same meaning of creating a pair (as opposed to "raw `and` JPEG", by the way).

> Suggestion: "when the same directory contains raw and non-raw images you may want to import only raws. all non-raws found can be ignored with this flag"
> 
> This could still be read as "when doing a multi-directory import, import all jpegs etc in a directory if there are no raws there". So an even simpler version could be:
> 
> "when both raw and non-raw images are found, this flag allows importing only the raws."

Thank you, in the morning with a fresh mind I'll choose the best of these suggestions.

Reply:
Yes, you are right; the suggestion was there before as well.